### DISCUSSION TITLE
1.3.0 버전에서 Zepeto World Multiplay가 보이지 않습니다.

### DISCUSSION THREAD

**Discussion by capgreen02** [naverz/zepeto-studio-kor#797] [5. Q&A]
원래 유니티에서 1.16 버전을 사용하고 있었습니다.
하지만, 테스트 도중 점프가 안된다는 걸 확인하고, 버전의 문제인가 싶어서 1.3.0 버전으로 업데이트하게 됐습니다.
그런 와중에 서버가 열리지 않아서 살펴보니 오브젝트에 넣었던 Zepeto World Multiplay가 사라졌습니다.

![문제22222](https://user-images.githubusercontent.com/98267406/170053852-96b3346d-b229-4126-9c47-30b901d40d30.png)

위와 같이 Component 창에서 multiplay를 검색해도 나오지 않습니다.
혹시 문제를 해결하는 방법을 알 수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. ASSETS > Reimport All 을 하신 후 다시 찾아보시기를 추천 드립니다.

**Reply by capgreen02** [reply]
감사합니다. 해결했습니다.
